sdcm/nemesis.py

```diff
@@ -126,15 +127,11 @@ def publish_event(self, disrupt, status=True, data=None):
     def set_current_running_nemesis(self, node):
         node.running_nemesis = self.__class__.__name__
 
-    def set_target_node(self):
-        """Set node to run nemesis on
-
-        Keyword Arguments:
-            is_running {bool} -- if True, method called upon nemesis start/running (default: {False})
-        """
-        filter_seed = self.cluster.params.get('nemesis_filter_seeds', default=True)
-
-        if filter_seed:
+    def set_target_node(self, node=None):
+        """Set node to run nemesis on"""
+        if node:
+            self.target_node = node
+        elif self.filter_seed:
```

Comment:
This change is not needed because you don't need to specifically remove. the nodes that were added.

Reply:
reverted back.